src/docfx/loc/LocConfigurationConvention.cs

```diff
@@ -10,17 +10,24 @@ internal static class LocConfigConvention
     {
         private static readonly Regex s_repoNameWithLocale = new Regex(@"^.+?(\.[a-z]{2,4}-[a-z]{2,4}(-[a-z]{2,4})?)?$", RegexOptions.IgnoreCase);
 
+        public static string GetEditRepository(this Config config, string locale)
```

Comment:
>GetEditRepository [](start = 29, length = 17)

What is the difference between GetEditRepository and GetLocRepository? If we put repository to input parameters, we can just expose this extension method?